### PR TITLE
Enable GRPA bandwidth estimator. Need GRPA binary from Intel if you set owt_use_grpa to true.

### PR DESCRIPTION
**Pull request by grgustaf**
This patch series allows integration with GPRA BWE library from Intel (library and header supplied externally).

### CODE REVIEW

File: modules/congestion_controller/send_side_congestion_controller.cc [hunk 1]
```diff
@@ -420,6 +429,21 @@ void SendSideCongestionController::OnTransportFeedback(
   DelayBasedBwe::Result result;
   {
     rtc::CritScope cs(&bwe_lock_);
+#if 1 // Debug purpose
```
Comment (taste1981) [inline modules/congestion_controller/send_side_congestion_controller.cc]: This debug log must be protected by INTEL_GRPA and by default turned off.

File: modules/congestion_controller/transport_feedback_adapter.cc [hunk 2]
```diff
@@ -89,6 +89,10 @@ void TransportFeedbackAdapter::SetNetworkIds(uint16_t local_id,
   remote_net_id_ = remote_id;
 }
 
+int64_t TransportFeedbackAdapter::GetCurrentOffsetMs() {
```
Comment (taste1981) [inline modules/congestion_controller/transport_feedback_adapter.cc:93]: if this API still exists, please protect in INTEL_GPRA macro.

File: modules/congestion_controller/transport_feedback_adapter.h [hunk 3]
```diff
@@ -55,6 +55,8 @@ class TransportFeedbackAdapter {
 
   size_t GetOutstandingBytes() const;
 
+  int64_t GetCurrentOffsetMs();
```
Comment (taste1981) [inline modules/congestion_controller/transport_feedback_adapter.h:59]: ditto

File: modules/congestion_controller/send_side_congestion_controller.cc [hunk 4]
```diff
@@ -459,8 +459,9 @@ void SendSideCongestionController::OnTransportFeedback(
         feedback_vector, acknowledged_bitrate_estimator_->bitrate_bps(),
         clock_->TimeInMilliseconds());
 #else
-	//Jianlin: current IncomingPacketFeedbackVector accepts 3 params instead of 1. Consider consolidate to upstream.
-    result = delay_based_bwe_->IncomingPacketFeedbackVector(feedback_vector);
+    result = delay_based_bwe_->IncomingPacketFeedbackVector(
```
Comment (taste1981) [inline modules/congestion_controller/send_side_congestion_controller.cc:460]: protect with GRPA macro.

Comment (grgustaf) [reply modules/congestion_controller/send_side_congestion_controller.cc:460]: This one is already, see #ifndef above.

**Comment by grgustaf** [comment]
Addressed all those; reorganized the one section for consistency.

**Comment by grgustaf** [comment]
One more update to turn off that debug message by default.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**